bin/cross-compile

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 set -ex
 
+cd exo-go
```

Comment:
This goes a couple layers deep. `bin/cross-compile` needs to be run at the root level so the `go-binaries/` directory exists on the root level as well. This used to be done in the `pre-publish` script defined in `package.json`, but since we're no longer running `yarn install` (per my comment above), we have to move `cross-compile` to the root level and run it there ourselves.